experimental/examples/bubblebench/editable-shared-tree/src/schema.ts

```diff
@@ -0,0 +1,234 @@
+                                                },
+                                            ],
+                                            vx: [
+                                                {
+                                                    type: numberSchema.name,
+                                                    value: 10,
+                                                },
+                                            ],
+                                            vy: [
+                                                {
+                                                    type: numberSchema.name,
+                                                    value: 10,
+                                                },
+                                            ],
+                                        },
+                                    },
+                                    {
+                                        type: iBubbleSchema.name,
+                                        fields: {
+                                            x: [
+                                                {
+                                                    type: numberSchema.name,
+                                                    value: 20,
+                                                },
+                                            ],
+                                            y: [
+                                                {
+                                                    type: numberSchema.name,
+                                                    value: 20,
+                                                },
+                                            ],
+                                            r: [
+                                                {
+                                                    type: numberSchema.name,
+                                                    value: 20,
+                                                },
+                                            ],
+                                            vx: [
+                                                {
+                                                    type: numberSchema.name,
+                                                    value: 20,
+                                                },
+                                            ],
+                                            vy: [
+                                                {
+                                                    type: numberSchema.name,
+                                                    value: 20,
+                                                },
+                                            ],
+                                        },
+                                    },
+                                ],
+                            },
+                        },
+                    ],
+                },
+            },
```

Comment:
Its unclear what is intended here. Looks like a list of nodes where the first is a client and the second is a list of clients? Seems wrong.

Maybe you just want a client?
```suggestion
```

Reply:
Yeah, this was left over from me trying two different ways to create a sequence in schema. This schema setup is modeled after `packages/dds/tree/src/test/feature-libraries/editable-tree/mockData.ts`. Perhaps want to take a look at that if there are issues here.